### ISSUE TITLE
Management of NVML init calls

### ISSUE THREAD

**Issue by asm582** [openshift/instaslice-operator#225]
There are three NVML init calls in the daemonset; there should only be one init call using setup, and the init should be destroyed when the manager is stopped. There are several ways to achieve this; one way is to have a separate manager file later registered with the controller call in the setup method.

**Comment by kannon92** [comment]
Is this issue still relevant?

**Comment by asm582** [comment]
Yes, the design is to create a file that performs nvml init once, reuses while the daemonset is up, and terminates when unavailable or deleted.

**Comment by asm582** [comment]
@rphillips, please correct me, but there was a PR from you to achieve the same. Can we close this issue?

**Comment by rphillips** [comment]
Yes, fixed with https://github.com/openshift/instaslice-operator/pull/327.

/close
